### DISCUSSION TITLE
Bootstrap 5 Admin Dashboard Theme -KaiAdmin

### DISCUSSION THREAD

**Discussion by techravi007** [oqtane/oqtane.framework#4648] [Q&A]
I'm revamping up my web development experience with Oqtane framework. 
I'm trying to create a Theme based on  bootstrap 5 Admin Dashboard Template from-
https://themewagon.com/themes/kaiadmin/

Though I'm following the instruction from below discussion and repository but not getting the expected result- 
https://github.com/oqtane/oqtane.framework/discussions/3919
https://github.com/oqtane/Oqtane.Theme.Arsha

Here is the result-
![image](https://github.com/user-attachments/assets/6aaf95f7-81b7-45b8-b485-f4bd3771433a)

Issues- 
        Admin Dashboard, Search bar and Login/Logout option not appearing.
        ModuleTitle, ModuleActions and ModuleInstance not loading.
        
Expectation-
  
![image](https://github.com/user-attachments/assets/1a0ae008-6622-4e57-a410-8a40c6d4b42f)

Can someone help me to figure out the issues?

Repository - https://github.com/techravi007/Techravindra.Theme.SkyeAdmin

Thanks for any help or suggestions.
🙏

**Comment by sbwalker** [comment]
The Arsha theme component (https://github.com/oqtane/Oqtane.Theme.Arsha/blob/main/Client/Themes/Default.razor) on line 23 contains:

```<UserProfile /> <Login /> <ControlPanel />```

Your theme component does not include these components.

**Reply by techravi007** [reply]
thanks @sbwalker 🙏
After putting missing components and default pane, It started working as expected.
